content/docs/gto/install.md

```diff
@@ -0,0 +1,33 @@
+# Installation
+
+To create an Artifact Registry with GTO, you only need a Git repo and GTO
+package installed. There's no need to set up any services or databases, compared
+to many other Model Registry offerings.
```

Comment:
```suggestion
You'll need [Python](https://www.python.org/) to install GTO, and
[Git](https://git-scm.com/) to use it. No special services or databases or
databases are required!
```

Reply:
Not clear now why you need DB/Services at all - if we talk about GTO installation, let's remove all mentions of MR.

Reply:
I removed the part about DBs because it didn't seem too relevant to mention in the installation page, but it may make sense in other docs.

Not sure I understood your suggestion wrt MR mentions.